unit-tests/live/options/test-presets.py

```diff
@@ -0,0 +1,51 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2023 Intel Corporation. All Rights Reserved.
+
+# test:device each(D400*) each(D500*)
```

Comment:
This might not work for 2 reasons.
1. not sure this syntax is supported, let's see
Normally I know we do:
```
#test:device each(D400*) 
#test:device each(D500*)
```
2. Not sure D500* will work, I think since we do not have D500 discovered LibCI might fail
If it happened, we need to remove it here and add it in another PR elsewere

Reply:
I will delete D500 for now.
I know we don't have D500 yet on LibCI but we will have soon. We will need to think on a solution to all the tests, adding D500 to all seems to much.